shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -385,7 +385,7 @@ public Options getOptions() {
     optEndRowExclusive.setArgName("end-exclusive");
     scanOptRow = new Option("r", "row", true, "row to scan");
     scanOptColumns = new Option("c", "columns", true,
-        "comma-separated columns.This" + " option is mutually exclusive with cf and cq");
+        "comma-separated columns.This option is mutually exclusive with cf and cq");
```

Comment:
Might as well fix this while in here:

```suggestion
        "comma-separated columns. This option is mutually exclusive with cf and cq");
```